rpmlb/__init__.py

```diff
@@ -1,25 +1 @@
-"""Top-level module
-"""
-
-import logging
-
 __version__ = '1.0.0'
```

Comment:
If we really want to have completely empty `__init__.py`, than yes. However, I'm perfectly happy with leaving the version string here.

As a side note, what about [setuptools_scm](https://pypi.python.org/pypi/setuptools_scm)? It seems to be more convenient that tracking the version in a string somewhere…